apparmor.d/groups/runit/runsv

```diff
@@ -0,0 +1,177 @@
+# apparmor.d - Full set of apparmor profiles
+# Copyright (C) 2019-2021 Mikhail Morfikov
+# Copyright (C) 2022 Alexandre Pujol <alexandre@pujol.io>
+# Copyright (C) 2024 Besanon <m231009ts@mailfence.com>
+# SPDX-License-Identifier: GPL-2.0-only
+
+abi <abi/3.0>,
+
+include <tunables/global>
+
+@{exec_pathrunsv} = @{bin}/runsv
+profile runsv @{exec_pathrunsv} flags=(attach_disconnected) {
+  include <abstractions/base>
+  include <abstractions/bus-session>
+  include <abstractions/bus-system>
+  include <abstractions/bus/org.freedesktop.login1>
+  include <abstractions/nameservice-strict>
+
+  capability fsetid,
+  capability fowner,
+  capability mknod,
+  capability chown,
+  capability setgid,
+  capability setpcap,
+  capability setuid,
+  capability fowner,
+  capability kill,
+
+  signal (send) peer=runsvdir,
+  signal (send) peer=runit,
+  signal (send) peer=NetworkManager,
+  signal (send) peer=agetty,
+  signal (send) set=(cont, term) peer=login,
+  signal (send) set=(cont, term) peer=dhcpcd,
+  signal (send) set=(cont, term) peer=sddm,
+  signal (send) peer=pause,
+  signal (send) peer=cupsd,
+  signal (send) peer=auditd,
+  signal (send) peer=chronyd,
+  signal (send) peer=dbus-daemon,
+  signal (send) peer=udevd,
+  signal (send) peer=rsyslogd,
+  signal (send) set=(kill, term) peer=startlxqt,
+  signal (send) set=(kill, term) peer=xorg,
+  signal (send) set=(term, cont) peer=@{p_systemd},
+  signal (receive) peer=runit,
+  signal (receive) peer=sddm,
+  signal (send) set=(cont, term) peer=elogind,
+  signal (receive) set=(cont, term) peer=elogind,
+  signal (receive) set=(hup) peer=@{p_systemd},
+
+  ptrace (read) peer=elogind,  
+  ptrace (read) peer=@{p_systemd},  
+  ptrace (trace) peer=@{profile_name},
+
+  @{exec_pathrunsv} 	 mr,
+  
+  @{bin}/sv 		 rPx,
+  @{bin}/vlogger         rPx,
+  @{bin}/udevadm 	 rCx -> udevadm, 
+  @{bin}/tlp             rPx,
+  @{bin}/readlink        rix,
+  @{bin}/ethtool         rix,
+  @{bin}/agetty          rPx,
+  @{bin}/id              rPx,
+  @{bin}/rsyslogd        rPx,
+  @{bin}/iw              rPx,
+  @{bin}/cupsd           rPx,
+  @{bin}/dhcpcd          rPx,
+  @{bin}/udevd           rPx,
+  @{bin}/dbus-daemon     rPx,
+  @{bin}/auditd          rPx,
+  @{bin}/chronyd         rPx,
+  @{bin}/NetworkManager  rPx,
+  @{bin}/mount  	 rPx,
+  @{bin}/sddm            rPx,
+  @{bin}/pause           rix,
+  @{bin}/install         rix,
+  @{bin}/chpst           rPx,
+  @{bin}/mkdir           rix,
+  @{bin}/mktemp          rix,
+  @{bin}/dbus-send       rix, # alt: rix
+  @{bin}/utmpset	 rix, # alt: rix
+  @{lib}exec/elogind/elogind          rPx,
+  @{lib}exec/elogind/elogind.wrapper  rPx,  # alt: rix
+  @{bin}/bash		 rPx, # alt: rix
+  @{bin}/tr		 rPx, # alt:rix
+  @{bin}/rm		 rix,
+  @{bin}/touch		 rix,
+  @{bin}/flock		 rix,
+  @{bin}/cat		 rix,
+  @{bin}/grep		 rPx, # alt:rix
+  @{bin}/mountpoint 	 rix,
+  @{bin}/systemctl       rCx -> systemctl,
```

Comment:
As the service manager, it can start anything:
```
  @{bin}/**                          PUx,
  @{lib}/**                           PUx,
  /usr/share/*/**                   PUx,
```

Reply:
Shouldn´t that be left for runit-init ?

Reply:
I don't know runit. Therefore I can't says which process is doing what. But looking at this list, to me it looks like the list of service started by your system.